internal/database/database.go

```diff
@@ -66,6 +75,9 @@ type DB interface {
 	// SaveAccount saves new account in database
 	SaveAccount(ctx context.Context, a model.Account) error
 
+	// SaveSettings saves settings for specific user in database
+	SaveSettings(ctx context.Context, a model.Account) error
```

Comment:
I'd use `SaveAccountSettings` for simplicity. I'm still not sure if we actually need a new method for this or if we could use the already existing `SaveAccount`.